evap/evaluation/tools.py

```diff
@@ -1,67 +1,15 @@
 import datetime
-import operator
-from collections import defaultdict
 from django.conf import settings
 from django.contrib.auth import user_logged_in
 from django.dispatch import receiver
 from django.utils import translation
 from django.utils.translation import LANGUAGE_SESSION_KEY, get_language
 
 
-# random object that will be used to check whether a default argument value was overwritten or not
-USE_DEFAULT = object()
-
-
 def is_external_email(email):
     return not any([email.endswith("@" + domain) for domain in settings.INSTITUTION_EMAIL_DOMAINS])
 
 
-def send_publish_notifications(evaluations, template_contributor=USE_DEFAULT, template_participant=USE_DEFAULT):
```

Comment:
import-outside-toplevel - I basically went through all the import statements that were nested somewhere and thought about how we could re-organize the code so that they become unnecessary. This one needed evaluation.models.EmailTemplate (obviously).